$OpenBSD: patch-t_spamd_ssl_v23_t,v 1.1 2013/10/16 22:29:50 bluhm Exp $

SSLv2 support has been removed from OpenSSL and Net::SSLeay

--- t/spamd_ssl_v23.t	Tue Jun  7 01:59:19 2011
+++ /dev/null	Wed Oct 16 21:54:20 2013
@@ -1,28 +0,0 @@
-#!/usr/bin/perl
-
-use lib '.'; use lib 't';
-use SATest; sa_t_init("spamd_sslv23");
-use Test; plan tests => (($SKIP_SPAMD_TESTS || !$SSL_AVAILABLE) ? 0 : 9);
-
-exit if ($SKIP_SPAMD_TESTS || !$SSL_AVAILABLE);
-
-# ---------------------------------------------------------------------------
-
-%patterns = (
-
-q{ Return-Path: sb55sb55@yahoo.com}, 'firstline',
-q{ Subject: There yours for FREE!}, 'subj',
-q{ X-Spam-Status: Yes, score=}, 'status',
-q{ X-Spam-Flag: YES}, 'flag',
-q{ X-Spam-Level: **********}, 'stars',
-q{ TEST_ENDSNUMS}, 'endsinnums',
-q{ TEST_NOREALNAME}, 'noreal',
-q{ This must be the very last line}, 'lastline',
-
-
-);
-
-ok (sdrun ("-L --ssl --ssl-version=sslv23 --server-key data/etc/testhost.key --server-cert data/etc/testhost.cert",
-           "--ssl=sslv23 < data/spam/001",
-           \&patterns_run_cb));
-ok_all_patterns();
